CONTRIBUTORS.txt

```diff
@@ -1,6 +1,64 @@
 The following people and organizations sponsored and/or contributed new and improved features to the project.
 
 ************************************************
+Key Contributors and Sponsors for 4.7
+************************************************
+
+CiviCRM Team - Atif Shaikh, Coleman Watts, David Greenberg, Eileen McNaughton,
+	       Jitendra Purohit, Josh Gowan, Kurund Jalmi,
+               Michael McAndrew, Monish Deb, Rohan Ramesh Katkar, Tim Otten,
+               Yashodha Chaku
+
+AGH Strategies - Andrew Hunt, Tyrell Cook, Nikki Murray
+Backoffice Thinking
+Cambridge University
```

Comment:
Hi, thanks for this. Would it be OK to change it to:
University of Cambridge - Alex Corr, John Kingsnorth

(then you can remove me from further down the list).